ci-operator/config/openshift/windows-machine-config-operator/openshift-windows-machine-config-operator-master.yaml

```diff
@@ -169,6 +169,13 @@ tests:
     - chain: upi-vsphere-windows-pre
     test:
     - ref: windows-unit-wicd
+- as: community-fast-forward
+  postsubmit: true
+  steps:
+    env:
+      DESTINATION_BRANCH: community-4.13
+      SOURCE_BRANCH: master
```

Comment:
Please mention why you are doing this for master instead of community-4.13.

Reply:
I couldn't add comments, I added this to the commit and PR description.

Reply:
Yes that is where I meant for it to no. No need for inline comments.